containers/podman-desktop
# feat: alert user when engine container actions fail

### What does this PR do?

This PR shows a short message next to the status in the resources page and add the full error text in the task view.

To keep it simple i didn't create any new component to show the full error, we can discuss what and where to create it here.
 
### Screenshot/screencast of this PR

![error-message-start](https://user-images.githubusercontent.com/49404737/228444181-f5bc4a3b-650a-461a-96c7-5420399cbc12.gif)

### What issues does this PR fix or reference?

this is part of #1337 
this fixes https://github.com/containers/podman-desktop/issues/1809

### How to test this PR?

1.  Just start a podman machine that cannot be started. I just brutally added the same error text as in the issue to see how a long message would look like in the task view.
